include/haywire.h

```diff
@@ -149,8 +149,9 @@ typedef struct
     int thread_count;
     char* parser;
     bool tcp_nodelay;
+    int max_request_size;
```

Comment:
Should this be unsigned? I'm sloppy about that too but we probably don't want negative sized buffers :P